.github/workflows/alpha.yaml

```diff
@@ -0,0 +1,23 @@
+name: alpha
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - run: make build
```

Comment:
To build it twice is by design ? I mean, aren't we building it through `deploy/Dockerfile` as well?

Reply:
nope. it was an oversight

Reply:
fixed